src/firefly/js/ui/tap/SpatialSearch.jsx

```diff
@@ -717,6 +718,9 @@ function makeSpatialConstraints(columnsModel, obsCoreEnabled, fldObj, uploadInfo
         const cenLat= cenLatField?.value;
         errList.checkForError(cenLonField);
         errList.checkForError(cenLatField);
+        if (!cenLon && !cenLat) errList.addError('Lon and Lat columns are not set');
```

Comment:
I'm not seeing this message displayed on the current data-int deployment; instead, I'm still seeing "Can not be empty.  Please provide value or expression"